templates/compose/signoz.yaml

```
@@ -0,0 +1,624 @@
+            <dictionaries_config>*_dictionary.xml</dictionaries_config>
+            <user_defined_executable_functions_config>*function.xml</user_defined_executable_functions_config>
+            <user_scripts_path>/var/lib/clickhouse/user_scripts/</user_scripts_path>
+            <http_port>8123</http_port>
+            <tcp_port>9000</tcp_port>
+            <mysql_port>9004</mysql_port>
+            <postgresql_port>9005</postgresql_port>
+            <interserver_http_port>9009</interserver_http_port>
+            <logger>
+              <level>information</level>
+              <formatting>
+                <type>json</type>
+              </formatting>
+            </logger>
+            <macros>
+              <shard>01</shard>
+              <replica>example01-01-1</replica>
+            </macros>
+            <prometheus>
+              <endpoint>/metrics</endpoint>
+              <port>9363</port>
+              <metrics>true</metrics>
+              <events>true</events>
+              <asynchronous_metrics>true</asynchronous_metrics>
+              <status_info>true</status_info>
+            </prometheus>
+            <opentelemetry_span_log>
+              <engine>engine MergeTree
+                      partition by toYYYYMM(finish_date)
+                      order by (finish_date, finish_time_us, trace_id)</engine>
+            </opentelemetry_span_log>
+            <query_masking_rules>
+              <rule>
+                <name>hide encrypt/decrypt arguments</name>
+                <regexp>((?:aes_)?(?:encrypt|decrypt)(?:_mysql)?)\s*\(\s*(?:'(?:\\'|.)+'|.*?)\s*\)</regexp>
+                <replace>\1(???)</replace>
+              </rule>
+            </query_masking_rules>
+            <send_crash_reports>
+              <enabled>false</enabled>
+              <anonymize>false</anonymize>
+              <endpoint>https://6f33034cfe684dd7a3ab9875e57b1c8d@o388870.ingest.sentry.io/5226277</endpoint>
+            </send_crash_reports>
+            <merge_tree_metadata_cache>
+              <lru_cache_size>268435456</lru_cache_size>
+              <continue_if_corrupted>true</continue_if_corrupted>
+            </merge_tree_metadata_cache>
+            <user_directories>
+              <users_xml>
+                  <!-- Path to configuration file with predefined users. -->
+                  <path>users.xml</path>
+              </users_xml>
+              <local_directory>
+                  <!-- Path to folder where users created by SQL commands are stored. -->
+                  <path>/var/lib/clickhouse/access/</path>
+              </local_directory>
+            </user_directories>
+            <default_profile>default</default_profile>
+              <distributed_ddl>
+                  <!-- Path in ZooKeeper to queue with DDL queries -->
+                  <path>/clickhouse/task_queue/ddl</path>
+              </distributed_ddl>
+          </clickhouse>
+
+  signoz:
+    image: signoz/signoz:latest
```

Please pin to a static version rather than 'latest' to prevent the template from breaking due to upstream changes.

```suggestion
  signoz:
    image: signoz/signoz:v0.97.1
```